include/nil/crypto3/zk/snark/proof.hpp

```diff
@@ -22,33 +25,73 @@
 // SOFTWARE.
 //---------------------------------------------------------------------------//
 
-#ifndef CRYPTO3_ZK_SNARK_PROOF_HPP
-#define CRYPTO3_ZK_SNARK_PROOF_HPP
-
-#include <nil/crypto3/zk/proof.hpp>
+#ifndef CRYPTO3_ZK_PLONK_PLACEHOLDER_PROOF_HPP
```

Comment:
This is #ifndef from include/nil/crypto3/zk/snark/systems/plonk/placeholder/proof.hpp file? They are not supposed to be the same...